src/main/java/org/opensearch/performanceanalyzer/metrics/PerformanceAnalyzerMetrics.java

```diff
@@ -33,17 +33,18 @@ public class PerformanceAnalyzerMetrics {
     public static final String sShardBulkPath = "shardbulk";
     public static final String sShardFetchPath = "shardfetch";
     public static final String sShardQueryPath = "shardquery";
-    public static final String sMasterTaskPath = "master_task";
+    public static final String sClusterManagerTaskPath = "cluster_manager_task";
     public static final String sFaultDetection = "fault_detection";
     public static final String sClusterApplierService = "cluster_applier_service";
-    public static final String sMasterClusterUpdate = "master_cluster_update";
+    public static final String sClusterManagerClusterUpdate = "cluster_manager_cluster_update";
     public static final String sElectionTermPath = "election_term";
     public static final String sHttpPath = "http";
     public static final String sOSPath = "os_metrics";
     public static final String sHeapPath = "heap_metrics";
     public static final String sNodesPath = "node_metrics";
     public static final String sPendingTasksPath = "pending_tasks";
-    public static final String sMasterThrottledTasksPath = "master_throttling_metrics";
+    public static final String sClusterManagerThrottledTasksPath =
+            "cluster_manager_throttling_metrics";
```

Comment:
nit : can be a single line
```suggestion
    public static final String sClusterManagerThrottledTasksPath = "cluster_manager_throttling_metrics";
```